packages/es-client/tests/test-executions.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
```

Comment:
Do we need an assertion here that reflects the new migrations? e.g. `... deletes executions with parents` or similar? Or is that covered elsewhere?

Reply:
I'm not sure if this comment is tied to the source directly or no - ES has no relational constraints, so 'with parents' is largely a distinction that doesn't matter.

Reply:
Oh these are es-client tests. Sorry, I saw the assertions and read them as endpoint/lib tests. Yeah agreed. Do we have tests elsewhere that set up execution parents and checks that the child execution successfully deletes?

Reply:
Hmm - that's a good point.   For ES - no.     This work *assumes* a lack of intra-collection situations and an intent to delete everything for the collection.  As such, it's a reasonable shortcut to just remove the records, however in the case where there *is* a execution record with a parent from another collection, you might have a bad parent record link in ES. 

It's probably worth a conversation re: that being a concern.

Reply:
That's right, the issue would be if you remove 'b', a will now hove an 'illegal' reference to 'a' in the parentArn field *in elasticsearch*    Happy to chat through it.